### PR TITLE
app-office/calcurse: add 4.8.1

### PR DESCRIPTION
**Pull request by kcning**

### CODE REVIEW

File: app-office/calcurse/calcurse-4.8.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+LICENSE="BSD-2"
+SLOT="0"
+KEYWORDS="~amd64 ~arm64 ~ppc ~ppc64 ~x86"
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+IUSE="caldav doc"
+
+RDEPEND="
+	sys-libs/ncurses:0=
+	elibc_musl? ( sys-libs/timezone-data )
```
Comment (arkamar) [inline app-office/calcurse/calcurse-4.8.1.ebuild:22]: This should not be conditional only for `elibc_musl` if calcurse requires it.

Comment (arkamar) [reply app-office/calcurse/calcurse-4.8.1.ebuild:22]: This is related to https://github.com/gentoo/gentoo/pull/31008, where I noticed it.

**Review by juippis — COMMENTED** [review]
Same error here, 
```
In file included from help.c:37:
calcurse.h:49:2: error: #error "Missing ncurses header. Aborting..."
   49 | #error "Missing ncurses header. Aborting..."
      |  ^~~~~
```

**Review by juippis — APPROVED** [review]
Thanks, in addition **virtual/pkgconfig** is needed but everything works now!

**Pull request closed without merging**